mobile/lib/widgets/settings/beta_timeline_list_tile.dart

```diff
@@ -83,21 +83,23 @@ class _BetaTimelineListTileState extends ConsumerState<BetaTimelineListTile> wit
             context: context,
             builder: (context) {
               return AlertDialog(
-                title: value ? const Text("Enable Beta Timeline") : const Text("Disable Beta Timeline"),
-                content: value
-                    ? const Text(
-                        "Are you sure you want to enable the beta timeline?",
-                      )
-                    : const Text(
-                        "Are you sure you want to disable the beta timeline?",
-                      ),
+                title: Text(
+                  value
+                      ? "advanced_settings_beta_timeline_enable_title".tr()
+                      : "advanced_settings_beta_timeline_disable_title".tr(),
+                ),
+                content: Text(
+                  value
+                      ? "advanced_settings_beta_timeline_enable_subtitle".tr()
+                      : "advanced_settings_beta_timeline_disable_subtitle".tr(),
+                ),
                 actions: [
                   TextButton(
                     onPressed: () {
                       context.pop();
                     },
                     child: Text(
-                      "cancel".t(context: context),
```

Comment:
have are transitioning to use the new `.t()` syntax so that it can handle plurals/singular easily